qualtran/cirq_interop/t_complexity_protocol.py

```diff
@@ -236,6 +239,9 @@ def _t_complexity_for_bloq(bloq: Bloq) -> Optional[TComplexity]:
     return _t_complexity_from_strategies(bloq, strategies)
 
 
+USE_NEW_GATE_COUNTING_FLAG = True
```

Comment:
Is this a temporary thing?

Reply:
yes

Reply:
follow-up PR will remove the vestigial code in this file, this flag, and `_t_comlpexity_` overrides